cirq-core/cirq/circuits/circuit_test.py

```diff
@@ -4652,11 +4652,16 @@ def test_create_speed():
     # Previously this took ~30s to run. Now it should take ~150ms. However the coverage test can
     # run this slowly, so allowing 2 sec to account for things like that. Feel free to increase the
     # buffer time or delete the test entirely if it ends up causing flakes.
+    #
+    # Updated in https://github.com/quantumlib/Cirq/pull/5756
```

Comment:
This looks like a performance regression. Could you file and issue and link to the issue instead of the PR? This way we're not going to loose track of the regression while avoiding the flaky failures for now.